packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -334,6 +334,7 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
 
     const loadBalancer = new ApplicationLoadBalancer(this, 'LB', {
       vpc: this.cluster.vpc,
+      vpcSubnets: props.vpcAlbSubnets ?? { subnetType: SubnetType.PRIVATE },
```

Comment:
Do we need to add the `?? { subnetType: SubnetType.PRIVATE }` part here? This matches the default behaviour, but why not let the constructor handle this defaulting itself?

Reply:
Don't *need* it. I just like being explicit.